udf-compiler/README.md

```diff
@@ -10,16 +10,10 @@ From `rapids-plugin-4-spark` root directory, use this command to run the `Opcode
 mvn test -DwildcardSuites=com.nvidia.spark.OpcodeSuite
 ```
 
-How to run spark shell
-----------------------
+How to run
+----------
 
-To run the spark-shell, you need a `SPARK_HOME`, the `cudf-0.15-cuda10-1.jar`, and the jars produced in the plugin. The cudf jar will be downloaded when mvn test (or package) is run into the ~/.m2 directory. It's easy to get the jar from this directory, and place somewhere accessible. In the case below, the cudf jar is assumed to be in a directory `$JARS`:
+The UDF compiler is included in the rapids-4-spark jar that is produced by the `dist` maven project.  Set up your cluster to run the RAPIDS Accelerator for Apache Spark
+and set the spark config `spark.sql.extensions` to include `com.nvidia.spark.udf.Plugin`.
```

Comment:
Not something for this PR, but should we file a followup to have the main Spark plugin auto-register the UDF plugin like it does the RAPIDS SQL plugin?  If there's a runtime master-enable like there is for the RAPIDS SQL plugin, that seems like a nicer user experience.

Reply:
oh I missed that, filed https://github.com/NVIDIA/spark-rapids/issues/688 to track. Seems that could be a different pr.